plugins/package-managers/node/src/main/kotlin/yarn2/WorkspaceInfo.kt

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2025 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.plugins.packagemanagers.node.yarn2
+
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.json.Json
+import kotlinx.serialization.json.decodeToSequence
+
+@Serializable
+internal data class WorkspaceInfo(
+    val location: String,
+    val name: String
+)
+
+private val JSON = Json { ignoreUnknownKeys = true }
```

Comment:
FYI, as of [kxs 1.8.0](https://github.com/Kotlin/kotlinx.serialization/releases/tag/v1.8.0) there's also a `@JsonIgnoreUnknownKeys` *annotation* that can be used on the concrete class where to ignore unknown values. This avoids the need to create an own mapper just for that purpose, and allows to use the same mapper both for classes where unknown keys should be ignored or not.